components/drv/ili9341.h

```diff
@@ -19,9 +19,9 @@ extern "C" {
  *      DEFINES
  *********************/
 #define DISP_BUF_SIZE (320 * 40)
-#define ILI9341_DC   19
-#define ILI9341_RST  18
-#define ILI9341_BCKL 23
+#define ILI9341_DC   2
+#define ILI9341_RST  4
+#define ILI9341_BCKL 21
```

Comment:
Why was it necessary to change the pins?

Reply:
I changed it to use my dev board, the one with only 30 gpios

Reply:
Altough, they are just randomly selected if there is no reason to change them please use the originals in this repo.

Reply:
Ok, i will fix them once i arrive home

Reply:
Fixed in 1140c49